python/lsst/ts/ATWhiteLightSource/wlsCSC.py

```diff
@@ -326,7 +339,8 @@ async def kiloarc_interlock_loop(self):
         """
         Make sure we stop the bulb if something bad happens with the chiller
```

Comment:
Does "stop the bulb" mean "turn off the light"? If so, please consider rewording for clarity. (I don't know what it means to "stop" a bulb).